3-UsingAPIs/Week2/homework/ex4-diceRace.js

```diff
@@ -15,18 +15,37 @@ const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
   const dice = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDie()
+  const promisesArray = dice.map((dice) => rollDie(dice));
```

Comment:
Plural: dice, singular: die

```suggestion
  const promisesArray = dice.map((die) => rollDie(die));
```

Or even simpler:

```js
const promisesArray = dice.map((die) => rollDie(die));
```